requirements.in

```diff
@@ -2,7 +2,7 @@ clamd==1.0.2
 Flask==2.3.2
 celery[sqs]==5.2.6
 Flask-HTTPAuth==4.8.0
-gunicorn==20.1.0
+gunicorn==22.0.0
```

Comment:
A little bit cautious on this one. The [changelog](https://docs.gunicorn.org/en/22.0.0/news.html) says

`HTTP versions < 1.0 or >= 2.0 are no longer accepted by default (rare, only HTTP/1.1 is supported)`

I think this says we have to send requests to gunicorn using HTTP/1.1 and can't send requests using HTTP/2

The bit that worries me is that our load balancers do have HTTP/2 enabled

<img width="175" alt="image" src="https://github.com/alphagov/notifications-antivirus/assets/7228605/2593f7c8-f3ff-4c2c-8030-7578e253f9ad">

You can see that on the Attributes tab for a given load balancer in our AWS account.

Technically, antivirus doesn't have a load balancer so there isn't HTTP/2 turned on, sending requests to the antivirus. This is because antivirus isn't available to the public internet and instead requests are sent directly between apps (not currently sure which version of HTTP they use). But if we were to merge this PR, it might be OK, but it also might not be (and we may not be able to do the equivalent for our other apps that do have external load balancers).

Or it might all be fine... depends somewhat on the internal workings of our AWS infra...

Reply:
requests is a lib specific to http 1.1 (first line of the docs - https://requests.readthedocs.io/en/latest/#)